tests/unit_tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from service.league_service.typedefs import League, LeagueDivision
+
+
+@pytest.fixture
+def example_league():
+    return League(
+        name="example_league",
+        divisions=[
+            LeagueDivision(
+                id=1, min_rating=None, max_rating=100, lowest_score=0, highest_score=10
```

Comment:
The keyword arguments aren't actually necessary, you could also write
```python
LeagueDivision(1, None, 100, 0, 10)
```
but I find that a lot less readable, similarly to the magic literals.

But if you're ever writing a lot of these, you might prefer the shorter syntax.